### PR TITLE
[T4A2][W15-B4]Jasper Lee

### PR DESCRIPTION
**Pull request by jasperlee27**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -9,13 +9,18 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
-
+    
+    public static int nextSequenceNumber = 1;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java:13]: this doesn't have to be public. in general, try to make all members private first, then protected, use public as last resort

**Comment by okkhoy** [comment]
you need to update the parser test file as well that uses the `parseCommand` method :-)
I think this is a good PR in general.

**Comment by okkhoy** [comment]
@jasperlee27 
Some comments added. Please ack & close the PR after reading comments.

**Comment by jasperlee27** [comment]
Acknowledged!

**Pull request closed without merging**